go.mod

```diff
@@ -3,8 +3,31 @@ module github.com/keptn-contrib/argo-service
 go 1.16
 
 require (
-	github.com/cloudevents/sdk-go/v2 v2.3.1
-	github.com/google/uuid v1.2.0
+	github.com/asaskevich/govalidator v0.0.0-20210307081110-f21760c49a8d // indirect
+	github.com/benbjohnson/clock v1.3.0 // indirect
+	github.com/cloudevents/sdk-go/observability/opentelemetry/v2 v2.8.0 // indirect
+	github.com/cloudevents/sdk-go/v2 v2.8.0
+	github.com/go-logr/stdr v1.2.2 // indirect
+	github.com/go-openapi/analysis v0.21.1 // indirect
+	github.com/go-openapi/runtime v0.21.0 // indirect
+	github.com/go-openapi/strfmt v0.21.1 // indirect
+	github.com/go-stack/stack v1.8.1 // indirect
+	github.com/google/uuid v1.3.0
+	github.com/hashicorp/golang-lru v0.5.4 // indirect
+	github.com/json-iterator/go v1.1.12 // indirect
 	github.com/kelseyhightower/envconfig v1.4.0
-	github.com/keptn/go-utils v0.8.0
+	github.com/keptn/go-utils v0.11.0
```

Comment:
Should we not use go-utils v0.9.x here?

Reply:
Yes, if we only use this for Keptn v0.9.x that makes sense. Another possibility would be to try out this repository version with Keptn version 0.10.x and 0.11.x and maybe use them for all, which would mean that we don't need three different releases.  

Let me know how you would like to proceed here.

Reply:
I'll accept it as it is, shouldn't be anything in there that stops us :)